dags/cas_ciip_portal_dags.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+"""
+# DAGs to deploy the ciip portal.
+ciip_deploy_db will initialize the portal database and import the swrs data
+cron_acme_issue will issue a certificate for the CIIP portal
+"""
+import json
+from dag_configuration import default_dag_args
+from trigger_k8s_cronjob import trigger_k8s_cronjob
+from airflow.operators.python_operator import PythonOperator
+from datetime import datetime, timedelta
+from airflow import DAG
+import os
+
+
+YESTERDAY = datetime.now() - timedelta(days=1)
+
+namespace = os.getenv('CIIP_NAMESPACE')
+
+ciip_deploy_db_args = {
+    **default_dag_args,
+    'start_date': YESTERDAY
+}
+
+SCHEDULE_INTERVAL = None
+
+default_dag = DAG('cas_ciip_portal_ciip_deploy_db', schedule_interval=SCHEDULE_INTERVAL,
+                  default_args=ciip_deploy_db_args)
+test_dag = DAG('cas_ciip_portal_ciip_deploy_db_test',
```

Comment:
I'm not sure if this DAG is still needed? If it is, it might be good to rename it something like `cas_ciip_portal_deploy_restore_from_prod` or something like that, so the user is not confused with what it does

Reply:
Yeah I'm also not sure if this dag is still used. @matthieu-foucault do you know if we use it?

Reply:
as @matthieu-foucault suggested we can have only one DAG that performs the "restore from prod" step if the `ENVIRONMENT` env variable is set to `test`